src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
```

Comment:
I would suggest using @getter and @setter in models too
We don't need toString() when we have fetchType.lazy depandancy. you might get LazyInitializationException when you call toString() on partly initialized product instance

Reply:
agree